examples/governance/methods/guest/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "governance-guest"
+version = "0.1.0"
+edition = "2021"
+
+[[bin]]
+name = "finalize_votes"
+path = "src/bin/finalize_votes.rs"
+
+[dependencies]
+k256 = { version = "=0.13.1", features = [
+  "arithmetic",
+  "serde",
+  "expose-field",
+  "std",
+  "ecdsa",
+], default-features = false }
+risc0-zkvm = { workspace = true, default-features = false, features = ["std"] }
+tiny-keccak = { version = "2.0", features = ["keccak"] }
+
+[profile.release]
+lto = "thin"
+opt-level = 3
+debug = true
```

Comment:
`opt-level = 3` is not required, as it is the default. Adding `codegen-units = 1` is generally helpful.
```suggestion
# Empirically observed to result in the best performance for this binary (in particular, better than
# "fat"). It's often a good idea to experiment with different optimization levels and options.
codegen-units = 1
lto = "thin"
debug = true
```